crates/core/src/db/queries/metadatas.rs

```diff
@@ -264,23 +269,38 @@ pub fn list(
         );
     }
 
-    if let Some(offerers) = offerers {
+    // ignoring the with_offers=false case for now, since it is more complicated and we dont currently have a need
+    let should_query_bid_receipts =
+        offerers.is_some() || (with_offers.is_some() && with_offers.unwrap_or(false));
+
+    if should_query_bid_receipts {
+        let mut bid_receipts_conditions = Condition::all().add(
+            Expr::tbl(BidReceipts::Table, BidReceipts::Metadata)
+                .equals(Metadatas::Table, Metadatas::Address),
+        );
```

Comment:
Isn't this always needed since its referenced on LN303?

Reply:
line 303 is under the should_query_bid_receipts block, so it will always be created when it is going to be used.